SmartPlaces.Facilities/lib/OntologyMapper.Mapped/src/Mappings/v1/Willow/mapped_v1_dtdlv2_Willow.json

```diff
@@ -960,7 +956,7 @@
     },
     {
       "InputDtmi": "dtmi:org:brickschema:schema:Brick:Break_Room;1",
-      "OutputDtmi": "dtmi:digitaltwins:rec_3_3:building:RestingRoom;1"
+      "OutputDtmi": "dtmi:com:willowinc:Room;1"
```

Comment:
This PR should be broken into multiple PRs. If we need to roll back the auth in a sperate project, it will roll back the model changes which is undesirable. We shouldn't have to write new code to revert breaks.